packages/internal-build-utils/src/dotnet.ts

```diff
@@ -58,3 +58,18 @@ export async function ensureDotnetVersionOrExit() {
 
   validatedDotnet = true;
 }
+
+/**
+ * Runs the dotnet formatter.
+ */
+export async function runDotnetFormat(...args: string[]) {
+  return runDotnetOrExit([
```

Comment:
Not for this PR, but I think it's about time to do something better than OrExit for skipping dotnet. It's too dangerous that somebody puts something after runDotnetFormat that shouldn't be skipped. We can keep propagating the OrExit up in names, but it's getting a little much.

Reply:
Maybe we should just require `dotnet` and only do the thing where we skip the VSIX part of the build without full VS.

Reply:
But like I said, not for this PR.

Reply:
Just requiring `dotnet` is much better than requiring visualstudio but still feels like an unnecessary requirement for most of Microsoft/cadl. For cadl-azure it might make sense to have `dotnet` required. But that doesn't help with the format helper.